eng/Versions.props

```diff
@@ -221,8 +221,8 @@
     <!-- ICU -->
     <MicrosoftNETCoreRuntimeICUTransportVersion>8.0.0-preview.2.23103.1</MicrosoftNETCoreRuntimeICUTransportVersion>
     <!-- MsQuic -->
-    <MicrosoftNativeQuicMsQuicVersion>2.1.1</MicrosoftNativeQuicMsQuicVersion>
-    <SystemNetMsQuicTransportVersion>7.0.0-alpha.1.22406.1</SystemNetMsQuicTransportVersion>
+    <MicrosoftNativeQuicMsQuicVersion>2.1.7</MicrosoftNativeQuicMsQuicVersion>
+    <SystemNetMsQuicTransportVersion>8.0.0-alpha.1.23106.6</SystemNetMsQuicTransportVersion>
```

Comment:
Does maestro not work anymore for this? I thought this should be flown automatically?
If we now update dotnet/msquic, will it get automatically propagated?

Reply:
we should check. But I did not want to wait for the cycle and it does not update MicrosoftNativeQuicMsQuicVersion

Reply:
it seems like maestro just did it.